python/docs/Makefile

```diff
@@ -1,19 +1,35 @@
 # Makefile for Sphinx documentation
 #
 
+ifndef SPHINXPYTHON
+SPHINXBUILD ?= sphinx-build
```

Comment:
If both are set, we prefer `SPHINXPYTHON`?

Reply:
yup, I updated the manual tests above. There are commands executed in the logs.